### PR TITLE
Sposor order

### LINKED ISSUE #511: Sponsor sorting in index page.
增加讓管理員能夠在後台排序贊助商顯示順序的功能.

### PR DESCRIPTION
**Pull request by Ritabear**
## Types of changes
**Thanks for sending a pull request! Please fill in the following content to let us know better about this change.**
Please put an `x` in the box that applies

- [x] **Bugfix**
- [ ] **New feature**
- [ ] **Refactoring**
- [ ] **Breaking change** (any change that would cause existing functionality to not work as expected)
- [ ] **Documentation Update**
- [ ] **Other (please describe)**

## Description

排order，但回傳的時候別回傳order，而是後台排好。

## Steps to Test This Pull Request

到Sponsor 的model 新增order=models.IntegerField(....) 
後台admin新增欄位
到view.py order_by 新增 'order' ----> order_by('level', 'order') 

## Expected behavior

回傳的API會依廠商相同地位照著order自己排好

## Related Issue

close #511

### CODE REVIEW

**Comment by Ritabear** [comment]
@mattwang44 
麻煩你看看了 ><

File: src/sponsors/models.py [hunk 1]
```diff
@@ -56,6 +56,12 @@ class Sponsor(ConferenceRelated):
             "fallback when the SVG file is not available."
         ),
     )
+    order = models.IntegerField(
+        verbose_name=_('order'),
+        max_length=10,
+        null=True,
+        blank=True,
+    )
```
Comment (mattwang44) [inline src/sponsors/models.py:64]: - `max_length` will not take any effect for the numeric field. If you'd like to constrain the range of the value, use [`validators`](https://docs.djangoproject.com/en/dev/ref/validators/#maxvaluevalidator) instead.
- Would using only positive value make more sense? (e.g. use `PositiveIntegerField` instead)
- IMO it's not easy to realize the purpose of this field simply by its name. Consider adding `help_text` here.
- What happened if there is any sponsor instance with this field to be null? Would it be at the start or the end of the returned list (and is that what you expected)?

**Review by mattwang44 — CHANGES_REQUESTED** [review]

**Comment by Ritabear** [comment]
@mattwang44  修改了~ 請幫我看看，感謝你！

**Comment by Ritabear** [comment]
@mattwang44 
修改好了~~ 麻煩matt看一下

**Review by mattwang44 — APPROVED** [review]
LGTM

加了一個 commit 是補 `order` 欄位在 admin site 上的翻譯

p.s. 自動產出 po 檔案的指令為
```
docker exec -it pycontw_dev sh -c "cd src && ./manage.py makemessages -l zh_Hant -l en_US"
```

**Pull request merged**